docs/jwst/cube_build/main.rst

```diff
@@ -28,6 +28,12 @@ Assumptions
 It is assumed that the ``assign_wcs`` step has been applied to the data, attaching the distortion and pointing
 information to the image(s). It is also assumed that the ``photom`` step has been applied to convert the pixel
 values from units of count rate to surface brightness. This step will only work with MIRI or NIRSpec IFU data.
+The cube_build algorithm is a flux conservation method and requires the input data to be in unit of surface brightness.
```

Comment:
I think I would say "... is a flux *conserving* method ...".  And towards the end of the line "unit" should be "units".